templates/hoover-deps.nomad

```diff
@@ -405,6 +405,67 @@ job "hoover-deps" {
   }
   {% endif %}
 
+  group "thumbnail-service" {
```

Comment:
Please surround this with an `if` because setting `count = 0` raises an error in Nomad, and so we can't avoid running this service that we want to make optional.